compiler/rustc_codegen_gcc/build_system/src/test.rs

```diff
@@ -677,7 +677,7 @@ fn test_projects(env: &Env, args: &TestArg) -> Result<(), String> {
 fn test_libcore(env: &Env, args: &TestArg) -> Result<(), String> {
     // FIXME: create a function "display_if_not_quiet" or something along the line.
     println!("[TEST] libcore");
-    let path = get_sysroot_dir().join("sysroot_src/library/core/tests");
+    let path = get_sysroot_dir().join("sysroot_src/library/coretests");
```

Comment:
Looks like https://github.com/rust-lang/rustc_codegen_gcc/pull/645/commits/73511151777a1a4b9404b430d3ac6b6444ccb2b4 is going to change this anyway. So maybe remove this change entirely to avoid conflicts?